packages/block-brokers/src/trustless-gateway/trustless-gateway.ts

```diff
@@ -19,6 +20,17 @@ export interface TrustlessGatewayComponents {
   transformRequestInit?: TransformRequestInit
 }
 
+export interface GetRawBlockOptions {
+  signal?: AbortSignal
+
+  /**
+   * The maximum number of bytes to allow when fetching a raw block.
+   *
+   * @default 2_097_152 (2MiB)
+   */
+  maxLimit?: number
```

Comment:
```suggestion
  maxSize?: number
```
it's maxSize in interface/blocks.. we should normalize.